pkg/client/run.go

```diff
@@ -121,6 +121,12 @@ func (c *SonobuoyClient) RunManifest(cfg *RunConfig, manifest []byte) error {
 				seenStatus = true
 			}
 
+			// if nil below was added for coverage on staticcheck
+			// TODO: ensure this is the desired behavior
+			if status == nil {
```

Comment:
Let's discuss this further.  Agreed we need to understand this is not adding new side effects.

Reply:
so after taking another look, it seems that in scenario where `status` is `nil`, previous behavior would have runtime de-referencing error on `switch{}`. so _prooooobably_ ok to have a more channeled error delivery?